src/ir/comp.rs

```diff
@@ -1280,6 +1309,8 @@ impl CompInfo {
                     let comment = cur.raw_comment();
                     let annotations = Annotations::new(&cur);
                     let name = cur.spelling();
+                    let is_public =
+                        cur.access_specifier() != clang_sys::CX_CXXPrivate;
```

Comment:
Why this? The handling of `protected` fields seems inconsistent with the code below.